test/sentry/opentelemetry/span_processor_test.exs

```diff
@@ -0,0 +1,126 @@
+defmodule Sentry.Opentelemetry.SpanProcessorTest do
+  use Sentry.Case, async: false
+
+  import Sentry.TestHelpers
+
+  alias Sentry.OpenTelemetry.SpanStorage
+
+  setup do
```

Comment:
If you start a server per test (`start_supervised` + not requiring a name) then this can be async and you won't need to do table cleanup.